kms/src/main/java/kms/DecryptAsymmetric.java

```diff
@@ -56,11 +61,41 @@ public void decryptAsymmetric(
       CryptoKeyVersionName keyVersionName =
           CryptoKeyVersionName.of(projectId, locationId, keyRingId, keyId, keyVersionId);
 
+      // Optional, but recommended: compute ciphertext's CRC32C. See helpers below.
+      long ceiphertextCrc32c = getCrc32cAsLong(ciphertext);
```

Comment:
This spelling error shows up in a few other places. I'd suggest doing a global find/replace
```suggestion
      long ciphertextCrc32c = getCrc32cAsLong(ciphertext);
```

Reply:
Thanks for catching that. Performed a global search/replace.